node/node_handler.go

```diff
@@ -248,6 +259,13 @@ func (node *Node) PostConsensusProcessing(newBlock *types.Block) {
 		node.BroadcastNewBlock(newBlock)
 	}
 	node.AddNewBlock(newBlock)
+
+	// TODO: enable drand only for beacon chain
```

Comment:
add comments here for your logic for the future review to understand your code & the future refactor

Reply:
sure.